packages/expo-notifications/src/__tests__/Notifications-test.ts

```diff
@@ -62,6 +62,29 @@ it(`verifies daily trigger handling`, async () => {
   );
 });
 
+it(`verifies weekly trigger handling`, async () => {
+  const trigger = {
+    weekday: 1,
+    hour: 12,
+    minute: 30,
+    repeats: true,
```

Comment:
```suggestion
    repeats: true as boolean | undefined,
```
This is going to make conflicts - with the upcoming update to TypeScript 4.0, calling
```ts
delete trigger.repeats;
```
will throw TypeScript error unless `repeats` is optional. See:
- #9960 (especially [this diff](https://github.com/expo/expo/pull/9960/files#diff-fb4f3cfd629d8eacd2eae220eff9244a)).
- https://devblogs.microsoft.com/typescript/announcing-typescript-4-0/#operands-for-delete-must-be-optional

Reply:
Thanks for your suggestion! Wasn't aware of this.